### DISCUSSION TITLE
Performance test issue

### DISCUSSION THREAD

**Discussion by mehrdad2000** [tilln/jmeter-iso8583#93] [General]
Hi
I have scenario that create message with iso8583 plugin (calculate field 35,64 with jrs223)
When start test wait around couple of seconds then seems once first create whole messages then send to destination.
Show calculated mac for whole message at log first then start to send.

while i thought per message should do this action!

Also send rate of transaction in monitoring dashboard but when i set “number of threads=1000” , “loop count infinite”, “duration 600” , and expect send 1000 message per second for 600 seconds but it send only 500 message that not continuously send to destination! Send 500 then 0 again 500 then 0, … and not continuously and lower than expected rate that i set that must be 1000. On time-chart show interrupted line.

Same behavior observe in lower rate like 50 tps.

any idea? 
thanks,

**Comment by tilln** [comment]
How much experience do you have using JMeter?
Sounds like you are struggling with JMeter scripting in general and not so much this plugin, so you may want to acquire some basic JMeter skills.
Though this is would not be the right forum.
